modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -0,0 +1,97 @@
+[id="installing-image-puller-using-che-operator_{context}"]
+= Installing {image-puller-name-short} using {prod-short} Operator
+
+Since version 7.24, it has been possible to use the {prod-short} Operator to install the {image-puller-name-short}. The Operator Spec has an `ImagePuller` field with the same parameters as the {image-puller-name-short} Operator.
```

Comment:
maybe let's remove the version reference since it will be harder to downstream

Reply:
True. We never want "since version <something>" statements (except in release notes).